packages/admin/admin/src/form/FinalFormContextProvider.tsx

```diff
@@ -5,12 +5,14 @@ export interface FinalFormContext {
     shouldScrollToField: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
     shouldShowFieldError: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
     shouldShowFieldWarning: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
+    shouldShowFieldHelperText: ({ fieldMeta }: { fieldMeta: FieldMetaState<any> }) => boolean;
```

Comment:
It's not optimal that this method has a different API in `Field` and `FinalFormContext`. This is a preexisting issue. I want to harmonize them in https://github.com/vivid-planet/comet/pull/1570 (targeting next).

But here we could do it "correctly" the first time and change this API to 

```ts
shouldShowFieldHelperText: (meta: FieldMetaState<FieldValue>) => boolean;
```

@johnnyomair What do you think?

Reply:
nvm, this method was removed